packages/components/src/components/domainproperties/DomainRowExpandedOptions.tsx

```diff
@@ -36,16 +37,23 @@ interface IDomainRowExpandedOptionsProps {
     appPropertiesOnly?: boolean
     domainIndex: number
     successBsStyle?: string
+    domainFormDisplayOptions?: IDomainFormDisplayOptions
 }
 
 export class DomainRowExpandedOptions extends React.Component<IDomainRowExpandedOptionsProps, any> {
```

Comment:
This component doesn't look to have the default prop set for the domainFormDisplayOptions. Do you think we should only have that at the top level DomainForm component and have it passed through everywhere or should we have the default everywhere (but I still think we'd need to pass it through all the way from DomainForm right)?